src/utils.js

```diff
@@ -65,6 +65,34 @@ export const newApifyClientErrorFromResponse = (statusCode, body, isApiV1) => {
     return new ApifyClientError(type, message, { statusCode });
 };
 
+/**
+ * Promisified version of request() package.
+ *
+ * @param  {String} method
+ * @param  {Object} options
+ * @return {Promise<[response, body]>}
+ */
+const promisifiedRequest = (method, options) => {
```

Comment:
Why not use `request-promise-native` package?

Reply:
Yeah good point, was there before we supported selected Promise implementation from options.